io/eolearn/io/sentinelhub_service.py

```diff
@@ -325,7 +342,10 @@ def __init__(self, layer, feature=None, **kwargs):
             feature = (FeatureType.DATA_TIMELESS, layer)
         elif isinstance(feature, str):
             feature = (FeatureType.DATA_TIMELESS, feature)
-        super().__init__(layer=layer, feature=feature, data_source=DataSource.DEM, **kwargs)
+        # if 'valid_data_mask_feature' not in kwargs:
+        mask_feature = (FeatureType.MASK_TIMELESS, 'IS_DATA_TIMELESS')
```

Comment:
You should put this as an init parameter: 
```Python
def __init__(self, layer, feature=None, valid_data_mask_feature=(FeatureType.MASK_TIMELESS, 'IS_DATA_TIMELESS'), **kwargs):
```
and then pass it on as such.
The problem in current version is that if user specifies different value for `valid_data_mask_feature` parameter it gets clashed with what you specify here by default.

Reply:
Am I missing something? The next line sets this precisely. Unfortunately, the comment is a bit misleading, so I'll remove it.

Reply:
Ah, now I see what you mean.

As this is `DEMWCSInput`, and as such a "helper" for the user, I would even go as far as not allow user to specify this parameter (e.g. removing from `**kwargs` if it exists). Will add as a default value in `__init__` for the time being.

Reply:
2 more things regarding this:
- `DEMWCSInput' now has this `valid_data_mask_feature` parameter, but `DEMWMSInput` doesn't. Should also `DEMWMSInput` have it?
- Why is this mask parameter necessary? I thought DEM is available globally and valid data mask would always have all True values?

Reply:
Probably, yes, will check and add if needed.

Reply:
I didn't add `valid_data_mask_feature` to `DEMWMSInput` as it didn't have it so far. In any case WMS is returning (typically) a 3-band jpg image, so I believe it is ok as is.

Reply:
The DEM we have at SentinelHub should be available everywhere, so you are right that it is probably not needed. 
Regarding the mask itself; I don't recall why I added it there. I'll remove it.